CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.2.1] - Unreleased
+### Fixed
+- Fixed a bug where the gem would only work with `Rails` and not all `ActiveRecord` projects
```

Comment:
Had this as a bug fix originally because before the `Rails.version` checks were put in place, this did work with non-rails applications, just had to manually require all the files. But I do like the idea of this being a fully fledged feature now so converted the version bump to a feature addition.